sycl/include/sycl/builtins.hpp

```diff
@@ -122,10 +122,27 @@ __SYCL_MATH_FUNCTION_OVERLOAD_FM(log2)
 __SYCL_MATH_FUNCTION_OVERLOAD_FM(log10)
 __SYCL_MATH_FUNCTION_OVERLOAD_FM(sqrt)
 __SYCL_MATH_FUNCTION_OVERLOAD_FM(rsqrt)
+__SYCL_MATH_FUNCTION_OVERLOAD_FM(fabs)
 
 #undef __SYCL_MATH_FUNCTION_OVERLOAD_FM
 #undef __SYCL_MATH_FUNCTION_OVERLOAD_IMPL
 
+template <typename T, size_t N>
+inline __SYCL_ALWAYS_INLINE
+    std::enable_if_t<detail::is_sgenfloat<T>::value, marray<int, N>>
+    ilogb(marray<T, N> x) __NOEXC {
+  marray<int, N> res;
+  for (size_t i = 0; i < N / 2; i++) {
+    vec<int, 2> partial_res =
+        __sycl_std::__invoke_ilogb<vec<int, 2>>(detail::to_vec2(x, i * 2));
+    std::memcpy(&res[i * 2], &partial_res, sizeof(vec<int, 2>));
```

Comment:
Good point actually, basically if it need more time to investigate, I suggest to implement all APIs as is right now to be conformant with SYCL 2020, and then took separate task to introduce more performant versions of some of these functions.

Reply:
@AlexeySachkov are you okay with Dima's suggestion? Can we just replace std::memcpy with detail::memcpy for now?

Reply:
Since we already have some built-ins implemented this way, I think it is okay to continue to follow it. We can do a bulk update later after an investigation. No need to do any changes right now